### DISCUSSION TITLE
0.4.1c

### DISCUSSION THREAD

**Discussion by alienatedsec** [alienatedsec/yi-hack-v5#316] [Announcements]
## What's Changed in `0.4.1c`
Changes by @alienatedsec 
* `wpa_supplicant` upgrade to `v2.10` - this is to prevent boot loop issues experienced with certain Wi-Fi configurations (e.g., multiple APs with the same SSID) or any modern Wi-Fi networks (e.g., mixed mode WPA/WPA2 - subject to be removed in the future `wpa_supplicant` releases)
* Should fix #312 and the boot loop issues discussed in the #291  
* Support for WPA3-Personal - Simultaneous Authentication of Equals (SAE)
* Support to encrypt your Wi-Fi password with `wpa_passphrase` - not tested as of yet

**Full Changelog**: https://github.com/alienatedsec/yi-hack-v5/compare/0.4.0...0.4.1c

**Comment by MichlW** [comment]
SNAPSHOT=yes
SNAPSHOT_LOW=no

These entries exist twice in system.conf from yi_outdoor_0.4.1c.tgz.

**Reply by alienatedsec** [reply]
Thanks @MichlW I found where it's duplicated.
